Sources/Paywalls/Events/PaywallEventsManager.swift

```diff
@@ -18,8 +18,10 @@ protocol PaywallEventsManagerType {
     @available(iOS 15.0, tvOS 15.0, macOS 12.0, watchOS 8.0, *)
     func track(paywallEvent: PaywallEvent) async
 
+    /// - Throws: if posting events fails
+    /// - Returns: the number of events posted
     @available(iOS 15.0, tvOS 15.0, macOS 12.0, watchOS 8.0, *)
-    func flushEvents(count: Int) async
+    func flushEvents(count: Int) async throws -> Int
```

Comment:
This new API makes it easier to assert (especially on integration tests) that things worked as expected.